Makefile

```
@@ -1,140 +1,88 @@
-DEFINES       += HAVE_BAGL_FONT_OPEN_SANS_REGULAR_11PX
-DEFINES       += HAVE_BAGL_FONT_OPEN_SANS_EXTRABOLD_11PX
-DEFINES       += HAVE_BAGL_FONT_OPEN_SANS_LIGHT_16PX
-DEFINES       += HAVE_UX_FLOW
-endif
+# Application allowed derivation curves.
+CURVE_APP_LOAD_PARAMS = secp256k1
 
-# Enabling debug PRINTF
-DEBUG = 0
-ifneq ($(DEBUG),0)
-
-        ifeq ($(TARGET_NAME),TARGET_NANOS)
-                DEFINES   += HAVE_PRINTF PRINTF=screen_printf
-        else
-                DEFINES   += HAVE_PRINTF PRINTF=mcu_usb_printf
-        endif
-else
-        DEFINES   += PRINTF\(...\)=
-endif
+# Application allowed derivation paths.
+PATH_APP_LOAD_PARAMS = "44'/888' 44'/1024'"
 
-##############
-#  Compiler  #
-##############
-ifneq ($(BOLOS_ENV),)
-$(info BOLOS_ENV=$(BOLOS_ENV))
-CLANGPATH := $(BOLOS_ENV)/clang-arm-fropi/bin/
-GCCPATH := $(BOLOS_ENV)/gcc-arm-none-eabi-5_3-2016q1/bin/
-else
-$(info BOLOS_ENV is not set: falling back to CLANGPATH and GCCPATH)
-endif
+# Setting to allow building variant applications
+VARIANT_PARAM = COIN
+VARIANT_VALUES = neo
 
-ifeq ($(CLANGPATH),)
-$(info CLANGPATH is not set: clang will be used from PATH)
-endif
+########################################
+#     Application custom permissions   #
+########################################
+# See SDK `include/appflags.h` for the purpose of each permission
+HAVE_APPLICATION_FLAG_BOLOS_SETTINGS = 1
+HAVE_APPLICATION_FLAG_GLOBAL_PIN = 1
 
-ifeq ($(GCCPATH),)
-$(info GCCPATH is not set: arm-none-eabi-* will be used from PATH)
-endif
+APPNAME = "NEO"
 
-CC := $(CLANGPATH)clang
-CFLAGS += -O3 -Os
+# U2F
+DEFINES   += HAVE_IO_U2F U2F_PROXY_MAGIC=\"NEO\"
+SDK_SOURCE_PATH  += lib_stusb lib_stusb_impl lib_u2f 
 
-AS := $(GCCPATH)arm-none-eabi-gcc
-AFLAGS +=
+########################################
+# Application communication interfaces #
+########################################
+ENABLE_BLUETOOTH = 1
 
-LD := $(GCCPATH)arm-none-eabi-gcc
-LDFLAGS += -O3 -Os
-LDLIBS += -lm -lgcc -lc
+########################################
+#         NBGL custom features         #
+########################################
+ENABLE_NBGL_QRCODE = 1
 
-APP_SOURCE_PATH += src
-SDK_SOURCE_PATH += lib_stusb lib_stusb_impl lib_u2f
-SDK_SOURCE_PATH += lib_ux
+# Use only specific files from standard app
+DISABLE_STANDARD_APP_FILES = 1
+APP_SOURCE_FILES += ${BOLOS_SDK}/lib_standard_app/io.c
+INCLUDES_PATH += ${BOLOS_SDK}/lib_standard_app
 
-ifeq ($(TARGET_NAME),TARGET_NANOX)
-SDK_SOURCE_PATH  += lib_blewbxx lib_blewbxx_impl
+ifeq ($(TARGET_NAME), TARGET_NANOS)
+DISABLE_STANDARD_BAGL_UX_FLOW = 1
 endif
 
-# Main rules
-
-all: default
-
-load: all
-	python -m ledgerblue.loadApp $(APP_LOAD_PARAMS)
-
-delete:
-	python -m ledgerblue.deleteApp $(APP_DELETE_PARAMS)
-
-# import rules to compile glyphs(/pone)
-include $(BOLOS_SDK)/Makefile.glyphs
-
-# Import generic rules from the SDK
-include $(BOLOS_SDK)/Makefile.rules
-
-listvariants:
-	@echo VARIANTS neo
+include $(BOLOS_SDK)/Makefile.standard_app
```

```suggestion
include $(BOLOS_SDK)/Makefile.standard_app

```
Missing empty line at the end of file?

---

Ok